hollow/src/main/java/com/netflix/hollow/tools/diff/HollowDiffMatcher.java

```diff
@@ -99,9 +99,11 @@ public void calculateMatches() {
             try {
                 matchedOrdinal = fromIdx.getMatchingOrdinal(key);
             } catch(NullPointerException ex) {
-                LOGGER.error("Null values found for type " + toTypeState.getSchema().getName()
+                LOGGER.log(Level.SEVERE, "Null values found for type " + toTypeState.getSchema().getName()
```

Comment:
So I actually agree with what you were saying earlier w.r.t there being no reason to log _and_ throw. Since we're going with rethrowing, how about we remove the log message entirely? As long as the message in the thrown RuntimeException has the details, we don't need to log here since the details will be included whenever the RuntimeException is logged or bubbles up.

Reply:
Sounds good.